assets/base.css

```diff
@@ -2227,22 +2231,39 @@ product-info .loading-overlay:not(.hidden) ~ *,
   line-height: calc(1 + 0.1 / var(--font-body-scale));
 }
 
-/* section-announcement-bar */
-.announcement-bar {
+/* utility-bar */
+.utility-bar {
+  height: 100%;
+}
+
+.utility-bar--bottom-border {
   border-bottom: 0.1rem solid rgba(var(--color-foreground), 0.08);
+}
+
+.announcement-bar,
+.announcement-bar-slider__announcement {
```

Comment:
Not a big deal, but I'd probably have the __announcement use the announcement-bar namespace (rather than announcement-bar-slider) `.announcement-bar__announcement`